self_test/self-test-manifest.json

```diff
@@ -0,0 +1,34 @@
+{
+  "manifest": {
+    "name": "Codius Test 200",
+    "version": "1.0.0",
+    "machine": "small",
+    "port": "80",
+    "containers": [
+      {
+        "id": "codius-test",
+        "image": "androswong418/codius-image-test@sha256:2ac5daaa973384203f206cc636d0a59c311f494215a2203f4a102d49289914e0",
+        "environment": {
+          "PORT": "$PORT",
+          "WEB_SOCKET": "$WEB_SOCKET"
+        }
+      },
+      {
+        "id": "nginx",
```

Comment:
It's possible to run a websocket server on the same port as an http server, so we probably don't need nginx in this pod.